wgpu/src/backend/mod.rs

```diff
@@ -1,9 +1,9 @@
 #[cfg(webgpu)]
-mod web;
+mod webgpu;
 #[cfg(webgpu)]
-pub(crate) use web::Context;
+pub(crate) use webgpu::{get_browser_gpu_property, ContextWebGpu};
 
-#[cfg(not(webgpu))]
-mod direct;
-#[cfg(not(webgpu))]
-pub(crate) use direct::Context;
+#[cfg(wgpu_core)]
+mod wgpu_core;
```

Comment:
This isn't a great name as it's a conflict I wonder if there's a better option

Reply:
maybe just `core`? I found the previous `direct` to be super confusing back in the day I figured this is a good opportunity to rename. Arguably we can also take this discussion separately

Reply:
changed your mind and just going with `wgpu_core` as module name despite the clash with the feature? 🤷 fine by me either way